zerver/management/commands/delete_realm.py

```diff
@@ -51,6 +52,14 @@ def handle(self, *args: Any, **options: str) -> None:
         if confirmation != realm.string_id:
             raise CommandError("Aborting!")
 
+        # Delete attachment files from the storage backend, so that we
+        # don't leave them dangling.
+        for obj_class in Attachment, ArchivedAttachment:
+            for path_id in obj_class.objects.filter(realm_id=realm.id).values_list(
+                "path_id", flat=True
+            ):
+                delete_message_attachment(path_id)
```

Comment:
Should we add something like this in `do_scrub_realm` as well?

Reply:
Ah, yes, absolutely.